src/PrestaShopBundle/Resources/config/services/core/grid/grid_data_factory.yml

```
@@ -129,3 +129,25 @@ services:
             - '@prestashop.core.hook.dispatcher'
             - '@prestashop.core.grid.query.doctrine_query_parser'
             - 'profiles'
+
+    prestashop.core.grid.data.factory.manufacturer:
+      class: 'PrestaShop\PrestaShop\Core\Grid\Data\Factory\DoctrineGridDataFactory'
+      arguments:
+        - '@prestashop.core.grid.query_builder.manufacturer'
+        - '@prestashop.core.hook.dispatcher'
+        - '@prestashop.core.grid.query.doctrine_query_parser'
+        - 'manufacturer'
+
+    prestashop.core.grid.data.factory.manufacturer_grid_data_factory:
```

Please follow the naming convention used by other similar services in this file, such as https://github.com/PrestaShop/PrestaShop/pull/9337/files#diff-73a0437f8671f28f736424c98011013dL119

```suggestion
    prestashop.core.grid.factory. manufacturer_decorator:
```